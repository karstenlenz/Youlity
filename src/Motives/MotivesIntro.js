import React from 'react'

export default function MotivesIntro() {
  return <></>
}
