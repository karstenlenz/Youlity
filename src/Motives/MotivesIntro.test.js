import { render } from '@testing-library/react'
import React from 'react'
import MotivesIntro from './MotivesIntro'
import { BrowserRouter as Router } from 'react-router-dom'

describe('The MotivesIntro component', () => {
  it('renders correctly', () => {
    const { container } = render(
      <Router>
        <MotivesIntro />
      </Router>
    )
    expect(container).toMatchSnapshot()
  })
})
