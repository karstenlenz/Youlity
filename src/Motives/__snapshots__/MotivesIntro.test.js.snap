// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The MotivesIntro component renders correctly 1`] = `
<div>
  <svg
    class="MotivesIntro__IntroImgStyled-sc-16bb2wg-0 hCpTvX"
    title=""
  >
    thinking.svg
  </svg>
  <section
    class="HeadlineUnderline__HeadlineContainer-sc-1reeb6v-1 tAJUt"
  >
    <div
      class="HeadlineUnderline__ChildContainer-sc-1reeb6v-0 cbzWsz"
    >
      <h1>
        Schritt 1: Bedürfnisse
      </h1>
    </div>
    <svg
      class="HeadlineUnderline__UnderlineStyled-sc-1reeb6v-2 dVDSEH"
    >
      underline.svg
    </svg>
  </section>
  <p>
    Die Persönlichkeit jedes Menschen ist von 6 Grundbedürfnissen bestimmt. Dabei gibt es bei jedem von uns eine unterschiedliche Gewichtung.
    <br />
    Wählen Sie in Schritt 1 Ihre drei wichtigsten Grundbedürfnisse.
    <br />
    In Schritt 2 werden Ihnen als Ergebnis Ihrer Auswahl zwei auf Sie angepasste Fragebögen präsentiert.
    <br />
    Im Anschluss erhalten Sie Ihr Ergebnis und dazu passende Informationen.
  </p>
  <section
    class="Collapse__CollapseWrapper-sc-1tgbx39-0 imOYVX"
  >
    <div
      class="Collapse__CollapseHeader-sc-1tgbx39-1 hKtGZf"
    >
      Mehr erfahren
      <img
        alt="auf/zuklappen"
        class="Collapse__CollapseIcon-sc-1tgbx39-2 bFSuxH"
        src="/img/expand.svg"
      />
    </div>
    <div
      class="Collapse__CollapseContent-sc-1tgbx39-3 domdvA"
    >
      Leben heißt, in Beziehung zu gehen. Wie Menschen in Beziehung gehen, hängt stark von ihrer Persönlichkeit ab. Einige Menschen halten Distanz, andere brauchen viel Nähe. Dabei spielen sechs Grundbedürfnisse eine zentrale Rolle, die schon in der Kindheit entstehen und häufig nicht bewusst sind. Welche Grundbedürfnisse für Menschen bedeutsam werden, entscheidet sich in ihrer individuellen Lebensgeschichte. Die Persönlichkeit jedes Menschen ist ein Ergebnis aus der Gewichtung dieser sechs parallel bestehenden und unterschiedlich ausgeprägten Grundbedürfnisse.
    </div>
  </section>
  <a
    class="FloatingButtonContainer__ButtonContainer-slkv4p-0 hTXSTP"
    href="/motives/entry"
  >
    <button
      class="Button__BtnStyled-sc-129yfz5-0 jyIaa-D"
      width="100"
    >
      Weiter zur Auswahl
    </button>
  </a>
  <div
    class="FloatingButtonContainer__BottomSpacer-slkv4p-1 hgDFeh"
  />
</div>
`;
